src/Tribe/JSON_LD/Event.php

```diff
@@ -108,11 +108,13 @@ protected function normalize_price( $price ) {
 			}
 		}
 
+		$locale_conv          = localeconv();
```

Comment:
Nice solution!

Thought for the future - maybe we also filter `$locale_conv[ 'mon_decimal_point' ]` for cases where the site admin can't influence `php.ini` for whatever reason (don't necessarily need that now though).